packages/pluggableWidgets/carousel-web/src/Carousel.editorPreview.tsx

```diff
@@ -0,0 +1,27 @@
+import { createElement, ReactNode, useCallback } from "react";
+import { CarouselPreviewProps } from "../typings/CarouselProps";
+import { Carousel } from "./components/Carousel";
+
+export function getPreviewCss(): string {
+    return require("./ui/Carousel.scss");
+}
+
+export function preview(props: CarouselPreviewProps): ReactNode {
+    return (
+        <Carousel
+            navigation={props.navigation}
+            pagination={props.showPagination}
+            loop={false}
+            items={[]}
```

Comment:
Not sure if there's any UI if there is no data, but we should include some dummy items here nonetheless. otherwise users in studio or using design preview can't interact with the widget

Reply:
The previous version was using dummy data, default image for that, but now @cjfhodges maybe can answer what we expect from preview?

Reply:
We had previous issue with Accordion that we didn't provide any data, which caused it to be unusable cuz users couldn't actually click it 😄  That's why we always just put some initial data, but yeah maybe Chris can help here

Reply:
I understand that, but before we do that I need to know what kind of design preview we are going to use, maybe it's just a simple SVG. That's why I asked Chris :)